### PR TITLE
Squash migrations

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

**Comment by mkurek** [comment]
remember to turn off docker image push and bintray image push on travis before merging this. And to warn people on gitter to not update source code for a few days until we create script for handling it.

File: src/ralph/settings/base.py [hunk 1]
```diff
@@ -95,7 +95,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': os.environ.get('DATABASE_NAME', 'ralph_ng'),
+        'NAME': os.environ.get('DATABASE_NAME', 'ralph_ng2'),
```
Comment (mkurek) [inline src/ralph/settings/base.py]: fix it

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
Closed (#1995 squash newest changes)

**Pull request closed without merging**